sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -91,6 +91,9 @@ platform. First, add the package:
 ```
 [//]: # ({x-version-update-end})
 
+- Known Issue: The pom.xml file should list `azure-messaging-servicebus` before `azure-identity` client libraries. 
```

Comment:
Can you edit this so it's clear that it's a known issue only with azure-identity:1.2.0 and earlier?  Since azure-identity:1.2.1 and above resolves this issue.